tests/subsys/settings/nvs/raw/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
```

Comment:
can you please use something else other than NONE.

Reply:
sure, I will correct it in the next patch